include/SFML/Graphics/Image.hpp

```diff
@@ -281,6 +281,12 @@ class SFML_GRAPHICS_API Image
     void flipVertically();
 
 private:
+    ////////////////////////////////////////////////////////////
+    /// \brief Directly initialize data members
+    ///
+    ////////////////////////////////////////////////////////////
+    Image(Vector2u size, std::vector<std::uint8_t>&& pixels);
```

Comment:
```suggestion
    explicit Image(Vector2u size, std::vector<std::uint8_t>&& pixels);
```

Not a big deal because `private`, but I'd still add it.